### PR TITLE
test: Adds BDD tests

### PR DESCRIPTION
**Pull request by soluchok**
Signed-off-by: Andrii Soluk <isoluchok@gmail.com>

### CODE REVIEW

File: test/bdd/fixtures/vct/mysql-config/mysql_config.sql [hunk 1]
```diff
@@ -0,0 +1,140 @@
+# MySQL / MariaDB version of the tree schema
```
Comment (troyronda) [inline test/bdd/fixtures/vct/mysql-config/mysql_config.sql:1]: should have origin url and license comments.

File: test/bdd/bddtests_test.go [hunk 2]
```diff
@@ -0,0 +1,123 @@
+package bdd
```
Comment (troyronda) [inline test/bdd/bddtests_test.go:1]: Missing copyright header.

Comment (soluchok) [reply test/bdd/bddtests_test.go:1]: I wonder why checks did not fail

File: test/bdd/fixtures/vct/mysql-config/mysql_config.sql [hunk 3]
```diff
@@ -0,0 +1,146 @@
+#
+# Copyright Google LLC, SecureKey Technologies Inc. All Rights Reserved.
```
Comment (troyronda) [inline test/bdd/fixtures/vct/mysql-config/mysql_config.sql:2]: Should adjust this line to link to the upstream project's copyright / license information. 

The next lines are good - URL of original file and SPDX license summary.

**Review by troyronda — APPROVED** [review]

**Pull request merged**